### PR TITLE
Update readme with p2p description

### PR DESCRIPTION
**Pull request by Roman-Halenko**
Added some info about what is p2p within the main course.

### CODE REVIEW

**Comment by zonzujiro** [comment]
Taking into action that p2p course can have no limits in time, I think we should better distinguish a description of the main and p2p course.

1. I think better will be is to place your part before `Admission`
2. Add information that the main course for 2019 is already have started and the only alternative - is p2p course. Then, after these lines, your text about what p2p course is.
3. Then, this line `You can use these materials for self-education as well.`
4. Also, I think it is can be a good idea to change the title of `Admission` to `Admission on the main course`

**Comment by Roman-Halenko** [comment]
@zonzujiro, Sure, completely agree

File: README.md [hunk 1]
```diff
@@ -18,9 +18,25 @@ You can use these materials for self-education as well.
...
+so fitted to strict deadlines as the main group is.
+
+The common factor of Peer teaching is that knowledge is shared not by an 
+instructor or other person of authority.
+It’s all about people on the same level teaching each other what they 
+know. __Learners Teaching Learners__. 
+
+As you could understand from the context of p2p meaning - it is a great
+possibility to grow up both as a student and teacher. Especially this course,
+you will learn how to correctly check the other students job.
```
Comment (zonzujiro) [inline README.md]: 🔥

**Comment by zonzujiro** [comment]
Great. I will merge it

**Pull request merged**